zer0-os/zOS
# Updates the Sendbird Client, and resolves the Sendbird Client not loading after page refresh

### What does this do?

Updates the Sendbird Client, and resolves the Sendbird Client not loading after page refresh

### Why are we making this change?

1. Sendbird was not loading after a page refresh even though the current user was authenticated
2. Sendbird [v4](https://github.com/sendbird/sendbird-chat-sdk-javascript) has better logging (add `logLevel: LogLevel.VERBOSE, debugMode: true` to the init params and observe), v4 has more bug fixes. Reference to the [migration guide](https://sendbird.com/docs/chat/v4/javascript/getting-started/migration-guide).

### How do I test this?

1. In MetaMask create a new address, connect the new address, verify you can create an account, refresh the page, verify that you are still connected to Sendbird (send a message)
4. Disconnect, then Connect as an existing user, refresh the page, verify you are still connected to Sendbird (send a message)
5. Sign in with a secondary account, send a message in a Messenger conversation, verify that real-time events are received 
6. Run tests

### Screen Capture

https://user-images.githubusercontent.com/31045/226956720-5a7a1591-5e94-44dc-9bb0-b9f641ad5c03.mov
